js/output/sql/sql-output.js

```diff
@@ -122,6 +122,39 @@ window.SQLOutput = Backbone.View.extend({
             errorMessage = i18n._("There's a syntax error near %(nearThing)s.",
                 {nearThing: nearPhrase.substr(5)});
         }
+        const colValueMismatchReg = /has (\d+) columns but (\d+) values were supplied/;
+        const colMismatchError = sqliteError.match(colValueMismatchReg);
+        if (colMismatchError) {
+          const tabName = sqliteError.split(" ")[1];
```

Comment:
Nit: I'd find this more readable if it was tableName, as I pronounced "tab" like a browser tab and got confused as to what it was talking about.
All of these rely on array indices existing, but I believe they're all guaranteed based on earlier matching, right?

Reply:
Done, thank you.